CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * [CHANGED] The experimental API for [finalizing payments on the server](https://stripe.com/docs/payments/finalize-payments-on-the-server?platform=android) has changed:
```

Comment:
Should we use `[BREAKING]` instead of `[CHANGED]` here?

Reply:
I don't think so, since it's beta, we don't want to scare users that are on the production only usage. But that's just my 2 cents.